Update docs with details on integration report (#370)
AlexsLemonade/scpca-downstream-analyses: optional-integration-analysis/README.md

2. **Integrate**: The merged data is integrated using [`fastMNN`](https://rdrr.io/github/LTLA/batchelor/man/fastMNN.html) and/or [`harmony`](https://portals.broadinstitute.org/harmony/articles/quickstart.html) to obtain batch corrected reduced dimensionality embeddings.
3. **Generate integration summary report**: Once the data from the multiple libraries are integrated and stored in the `SingleCellExperiment` object, the results from batch correction are evaluated to create a summary report.
This report includes UMAPs showing data from all libraries before and after batch correction and summaries of a set of integration metrics.
The plots are displayed in a html report for ease of reference.

The metrics we use to evaluate integration, as mentioned in step 3 above, include:

> - Batch average silhouette width (ASW), which is used to assess consistency within clusters and measures how well a given data point adheres to its own cluster vs. other clusters.

Reviewer: ```suggestion
- Batch average silhouette width (ASW), which assesses consistency within clusters and measures how close a given data point adheres to cells from the same cluster vs. other clusters.
```
